package.json

```diff
@@ -16,7 +16,8 @@
     "coreUpgrade": "mvn test -Dcucumber.filter.tags='@selenium and @initialSetup and @upgrade'",
     "ocl": "./node_modules/.bin/cypress-tags run -eTAGS='@cypress and @ocl'",
     "microfrontend": "./node_modules/.bin/cypress-tags run -eTAGS='@cypress and @microfrontend'",
-    "refApp3Login": "cypress run --spec src/test/resources/features/refapp-3.x/01-login/login.feature"
+    "refApp3Login": "cypress run --spec src/test/resources/features/refapp-3.x/01-login/login.feature",
```

Comment:
Is this even needed to track the automations for RefApp 3.x?

Reply:
Here Git makes it to be tracked as a change when seemingly is not actually a change. To add a command that triggers a new test, one needs to add a comma `,` on the previous command within the `scripts` section. Adding a comma on the existing command `which-ever` makes git to track it as a change on that command.